plugins/servicenow/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: ["insightconnect"]
 name: servicenow
 title: ServiceNow
 description: ServiceNow is a tool for managing incidents and configuration management. Using the ServiceNow plugin for Rapid7 InsightConnect, users can manage all aspects of incidents including creation, search, updates, as well as monitor them for changes
-version: 7.3.1
+version: 7.4.0
 connection_version: 7
 supported_versions: ["2020-03-11 Orlando"]
```

Comment:
Can you confirm if this was the version you completed your testing on?

Reply:
Hey Dympna - I have been using New York for testing.